lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -79,13 +75,11 @@ sub edit {
     $self->redirect_to('edit_step', moduleid => $running_module->name(), stepid => $stepid);
 }
 
-sub streamtext {
-    my ($self, $file_name, $start_hook, $close_hook) = @_;
-
+sub streamtext ($self, $file_name, $start_hook, $close_hook) {
```

Comment:
`Too few arguments for subroutine 'OpenQA::WebAPI::Controller::Running::streamtext' at t/26-controllerrunning.t line 76.`